examples/11-refund-payment.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+#
+# Example 11 - How to prepare a new refund with the Mollie API.
+#
+from __future__ import print_function
+
+import os
+
+from mollie.api.client import Client
+
+
+def main():
+
+    #
+    # Initialize the Mollie API library with your API key.
+    #
+    # See: https://www.mollie.com/dashboard/settings/profiles
+    #
+
+    api_key = os.environ.get('MOLLIE_API_KEY', 'test_test')
+    mollie_client = Client()
+    mollie_client.set_api_key(api_key)
+
+    body = ''
+    payment_id = ''
+
+    payments = mollie_client.payments.all()
+
+    body += '<p>No payment ID specified. Attempting to retrieve all payments and grabbing the first.</p>'
```

Comment:
Dit suggereert dat je wel een payment id kan specificeren, bijvoorbeeld via een querystring parameter. Dat is echter helemaal niet mogelijk.